core/src/border.rs

```diff
@@ -54,7 +54,7 @@ impl Border {
 /// The border radii for the corners of a graphics primitive in the order:
 /// top-left, top-right, bottom-right, bottom-left.
 #[derive(Debug, Clone, Copy, PartialEq, Default)]
-pub struct Radius([f32; 4]);
+pub struct Radius(pub [f32; 4]);
```

Comment:
I don't think this needs to be `pub` — the `from()` methods can be used.

Reply:
Sorry, you are right. Fixed.